app/forms/claim_form.rb

```diff
@@ -0,0 +1,45 @@
+class ClaimForm
+  include ActiveModel::Model
+  STEPS = %w[providers mentors].freeze
```

Comment:
No change needed at this point. But in the future, I could imagine you wanting to have one Form Object per 'step' of the Claim form.

Otherwise I could imagine this class becoming huge and conditionally doing different things depending on the current step.

Reply:
Yeah, good point. At this point it didn't feel necessary